### DISCUSSION TITLE
Unable to retrieve any data from API

### DISCUSSION THREAD

**Discussion by DGBooth** [wolveix/satisfactory-server#382] [General]
I am attempting to write a script which checks the current steamcmd version against the currently running server version, and restarts the container if no players are active and the server version is out of date.

When querying the api I am only getting the following response
```
curl -k -s "https://127.0.0.1:7777/"
{"errorCode": "errors.com.epicgames.httpserver.route_handler_not_found", "errorMessage": ""}
```

Have attempted:
- 127.0.0.1
- localhost
- local ip
- from inside the container
- from outside, but on localhost
- from outside, from my main pc

From the server logs, seems to be listening on port 7777

Any ideas, or if this problem already has a solution I'd love to be pointed in the right direction.

**Comment by wolveix** [comment]
https://github.com/wolveix/satisfactory-server/wiki/Official-API-Docs
